### PR TITLE
chore(examples): reordering folders

### PR DESCRIPTION
**Pull request by ozkalai**
in this PR, We have reordered the examples' folders separately as Ant Design ( `antd` ) and Material UI ( `mui` ).

### CODE REVIEW

File: documentation/blog/2021-10-6-migrating-react-admin-application-to-refine.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ import react_admin_create from '@site/static/img/blog/2021-10-6-migrating-react-
 Recently, our team decided to migrate a B2B admin panel of one of our customers from React-Admin to refine to battle test our new framework and improve our productivity. My mission was to migrate it and it took one and a half days for me to rewrite the panel in refine.
 
 <!--truncate-->
+
 ### Migrating a React-Admin Application to refine
 
 [refine](https://refine.dev/) is an extremely customizable [Ant Desig](https://ant.design/) based [React](https://reactjs.org/) framework for data-intensive applications and it handles most of the CRUD functionality that can be a requirement in many applications, without much effort. Providing the rest of the desired functionality (outside CRUD) is up to you, like in any <strong>React</strong> application.
```
Comment (aliemir) [inline documentation/blog/2021-10-6-migrating-react-admin-application-to-refine.md:22]:
```suggestion
[refine](https://refine.dev/) is an extremely customizable [Ant Design](https://ant.design/) based [React](https://reactjs.org/) framework for data-intensive applications and it handles most of the CRUD functionality that can be a requirement in many applications, without much effort. Providing the rest of the desired functionality (outside CRUD) is up to you, like in any <strong>React</strong> application.
```

**Review by aliemir — APPROVED** [review]

File: examples/importExport/mui/src/App.tsx [hunk 2]
```diff
@@ -4,32 +4,37 @@ import {
...
-            catchAll={<ErrorComponent />}
-            resources={[
-                {
-                    name: "posts",
-                    list: ImportList,
-                },
-            ]}
-        />
+        <ThemeProvider theme={LightTheme}>
+            <CssBaseline />
```
[the remark this answers is not in the document]
Comment (ozkalai) [reply examples/importExport/mui/src/App.tsx:21]: I think there is no need to add font smoothing here, I checked the sider collapse behavior and it works properly.

**Comment by omeraplak** [comment]
Hey @ozkalai,
Can we move "baseHeadless" folder to "headless" in "base"?

File: examples/importExport/mui/src/App.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (omeraplak) [reply examples/importExport/mui/src/App.tsx:20]: Is `WebkitFontSmoothing` related to Sider?

https://github.com/pankod/refine/blob/next/examples/base/mui/src/App.tsx#L23

[the remark this answers is not in the document]
Comment (ozkalai) [reply examples/importExport/mui/src/App.tsx:20]: I can't see any difference between auto smoothing and without it.

File: examples/importExport/mui/src/pages/list.tsx [hunk 3]
```diff
@@ -27,7 +44,17 @@ export const ImportList: React.FC = () => {
...
+    const { open } = useNotification();
+
+    const { inputProps, isLoading } = useImport({
+        onFinish: () => {
+            open &&
+                open({
+                    message: "Import successfully completed",
+                    type: "success",
+                });
+        },
```
Comment (omeraplak) [inline examples/importExport/mui/src/pages/list.tsx:56]:
```suggestion
            open?.({
                    message: "Import successfully completed",
                    type: "success",
                });
        },
```

**Review by omeraplak — CHANGES_REQUESTED** [review]

**Pull request merged**